test/parallel/test-fs-watchfile-ref-unref.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const common = require('../common');
+
+const fs = require('fs');
+const assert = require('assert');
+
+let refCount = 0;
+
+const watcher = fs.watchFile(__filename, common.mustNotCall());
+
+function unref() {
+  watcher.unref();
+  refCount--;
```

Comment:
Not quite sure I get what purpose the `refCount--` accounting is for here?

Reply:
Just to keep track of the number of calls, I'll delete these.

Reply:
fixed